nestkernel/archiving_node.cpp

```diff
@@ -163,22 +163,22 @@ nest::Archiving_Node::get_history( double t1,
   }
   else
```

Comment:
I would suggest to remove the `else` here. Since the `if` ends with a return, we don't really need it.